### PR TITLE
Message linking client server

### PR DESCRIPTION
**Pull request by FlXlT**
From Client point of view:
- added a button on which you can press to send a message to the server
- fixes JavaDoc.

To Be Done: Static Errors
Server: https://github.com/jessetilro/pixelperfect/pull/118

### CODE REVIEW

File: app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/GameActivity.java [hunk 1]
```diff
@@ -2,12 +2,45 @@
...
+     *
+     * @param savedInstanceState , a Bundle.
+     */
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_game);
+
+        Button button = (Button) findViewById(R.id.button_complete_event);
+        button.setOnClickListener(this);
```
Comment (jessetilro) [inline app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/GameActivity.java:30]: This implementation will not work if we have multiple buttons. You can specify the method that must be called in the activity for each button in the XML. I suggest we use this approach. (Take a look at the main activity and its XML sheet, it is done there.)

File: app/src/main/java/nl/tudelft/pixelperfect/client/ClientListener.java [hunk 2]
```diff
@@ -5,14 +5,19 @@
...
 
 /**
+ * The ClientListenere waits for incoming messages from the server and interps them.
+ *
  * @author Jesse Tilro
+ * @author Floris Doolaard
  */
 public class ClientListener implements MessageListener<Client> {
     public void messageReceived(Client source, Message message) {
         if (message instanceof HelloMessage) {
```
Comment (jessetilro) [inline app/src/main/java/nl/tudelft/pixelperfect/client/ClientListener.java:15]: Maybe we should remove the entire HelloMessage in both client and server, it was just temporary for testing purposes.

File: app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/GameActivity.java (hunk 1 above)
Comment (FlXlT) [inline app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/GameActivity.java:30]: Alright, I will use this in my next Issue.

File: app/src/main/java/nl/tudelft/pixelperfect/client/ClientListener.java (hunk 2 above)
Comment (FlXlT) [inline app/src/main/java/nl/tudelft/pixelperfect/client/ClientListener.java:15]: Correct.

**Comment by dalderliesten** [comment]
Since you commented on the XML button issue, looks fine to me.

**Pull request merged**